Plutarch/Api/V1.hs

```diff
@@ -116,7 +116,7 @@ redeemerHash :: Plutus.Redeemer -> Plutus.RedeemerHash
 redeemerHash = coerce . dataHash
 
 -- | Hash the data encoded representation of given argument.
-dataHash :: Plutus.ToData a => a -> Plutus.BuiltinByteString
+dataHash :: (Plutus.ToData a) => a -> Plutus.BuiltinByteString
 dataHash = hashData . Plutus.toData
```

Comment:
Add `single-constraint-parens: never` to fourmolu.yaml

Reply:
Neat. Thanks